Project.toml

```diff
@@ -4,3 +4,18 @@ authors = ["Philip Fackler <facklerpw@ornl.gov>",
            "Steven Hahn <hahnse@ornl.gov>",
            "William F Godoy <godoywf@ornl.gov>"]
 version = "0.1.0"
+
+[deps]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
```

Comment:
No need to add CUDA explicitly if using JACC :)

Reply:
oh, has something changed since last we spoke?

Reply:
Nope, they don't need to be in the Project.toml in the repo. This gets added when running cases due to Pkg.add action polluting Project.toml, but we shouldn't be locking to one vendor or keep it in the repo. If that makes sense. See my other comment for an example.

Reply:
@williamfgc  I have done as you suggested

Reply:
Thanks @PhilipFackler, I wish I knew a better way to hide back ends as much as possible.